js/ui/camera.js

```diff
@@ -411,7 +413,8 @@ class Camera extends Evented {
      * details not specified in `options`.
      *
      * @memberof Map#
-     * @param {CameraOptions|AnimationOptions} options Options describing the destination and animation of the transition.
+     * @param {Object} options Options describing the destination and animation of the transition.
+    *            Accepts [CameraOptions](#CameraOptions) and [AnimationOptions](#AnimationOptions).
```

Comment:
That seemed slightly misleading, because the object doesn't have to be *either* a CameraOptions *or* an AnimationOptions, but can share attributes from both.